src/main/java/com/spotinst/sdkjava/model/EbsDevice.java

```diff
@@ -240,5 +284,20 @@ public boolean isVolumeTypeSet() {
     public boolean isThroughputSet() {
         return isSet.contains("throughput");
     }
+
+    // Is dynamicIops Set boolean method
+    @JsonIgnore
+    public boolean isDynamicIopsSet() {
+        return isSet.contains("dynamicIops");
+    }
+
+    // Is kmsKeyId Set boolean method
+    @JsonIgnore
+    public boolean isKmsKeyIdSet() {
+        return isSet.contains("kmsKeyId");
+    }
+
```

Comment:
"isDynamicVolumeSizeSet" block is missing

Reply:
No It is there 
![Uploading image.png…]()